src/App.vue

```diff
@@ -1106,6 +1102,19 @@ export default class App extends Vue {
 
         // fetch the data
         await this.fetchData()
+
+        // show survey dialog...
+        // - if this is an Incorporation Application filing
+        // - if the IA Survey ID is configured
+        // - if the cookie doesn't exist (eg, never set, or expired)
+        if (this.isIncorporationFiling) {
+          if (sessionStorage.getItem('IA_SURVEY_ID')) {
+            const cookie = Cookies.get(this.IA_SURVEY_KEY, { domain: this.domain })
+            if (!cookie) {
+              this.filingSurveyDialog = true
+            }
+          }
+        }
```

Comment:
Moved here as this block is called only once, whereas fetchData() (where this was before) could be called additional times to retry from an error dialog.